sdk/translation/azure-ai-translation-document/tests/test_translation_async.py

```diff
@@ -463,8 +463,8 @@ def callback(request):
             assert input.source.filter.prefix == ""
             assert input.source.filter.suffix == ".txt"
             assert input.storage_type == "File"
-            assert input.targets[0].category == "fake"
-            assert input.targets[0].glossaries[0].format == "txt"
+            assert input.targets[0].category_id == "fake"
```

Comment:
These two attributes here are breaking API changes?

Reply:
Actually it seems in the APIview diff that these two properties were always called `category_id` and `file_format`?

Reply:
Sorry it's a bit confusing - we renamed a lot of things in the GA version of this library. `StartTranslationDetails` (the type we're accessing here) was not exposed in the GA library. We didn't expose this in the beta release following the GA either, but it was used here in a test to verify deserialization.

In this PR, we make StartTranslationDetails public, as another input option to begin_translation. To be consistent with the renaming we did in the GA library, I've renamed these two properties to match.